scripts/build.js

```diff
@@ -46,6 +47,49 @@ function afterBuild(options) {
 }
 
 /**
+ * initSubmodules
+ *
+ * @param {Function} cb
+ * @api private
+ */
+
+function initSubmodules(cb) {
+  var errorMsg = '';
+  var git = spawn(['LIBSASS_GIT_VERSION=', pkg.libsass, ' ./scripts/git.sh'].join());
```

Comment:
Make sure we checkout out libSass at the version we're reporting to user via `node-sass -v`.

This also gives us a central hook if we're experimenting with libSass branches etc..